### ISSUE TITLE
[PSPE] TNT explosions look bad

### ISSUE THREAD

**Issue by magicus** [ProtocolSupport/ProtocolSupportPocketStuff#92]
When you ignite a TNT block, it flashes white and jumps (as it should), but then it just disappears, nothing seem to happen, and then suddenly you get an update with a lot of blocks missing. There are no explosion effect. If I remember correctly, the TNT block should flash white for a few seconds before exploding. Now it just disappears.

**Comment by 7kasper** [comment]
Yeah this is probs some id that changed. Also something that worked fine before.

**Comment by magicus** [comment]
I've looked a bit more closely what's wrong. It's two things:
1) The "primed TNT" entity does not appear. I do believe this is a regression from before, but I'm not 100% sure (and can't check right now)

2) The particle effect after the explosion is not completely correct. There is some "white/gray circles" thing, which is correct, but when running locally on PE I also get some lingering effects, like smoke from a torch. This is missing.

3) Something just looks odd about it; it might be a timing effect between the particle effect appearing and the blocks disappearing. Maybe when running locally, the blocks disappear in expanding circles from the source of the explosion. I think it looks like that, but it's hard to tell. Anyway, the PSPE explosions look more "instant" and artificial.

I think 3) might be hard to fix at all, 2) could be related to other particle effect issues, and 1) might be an instance of general issues with entities.
